src/content/release/breaking-changes/android-surface-plugins.md

```diff
@@ -3,15 +3,16 @@ title: New APIs for Android plugins that render to a Surface
 description: >
   Adds a new API, `SurfaceProducer`, to the Android embedding API, which
   opaquely handles the creation and management of a `Surface` for plugins. For
-  Impeller, use of this API is required.
+  [Impeller][], use of this API is recommended.
 ---
 
 ## Summary
 
 The Android embedder for Flutter introduces a new API, [`SurfaceProducer`][],
 which allows plugins to render to a `Surface` without needing to manage what
 the backing implementation is. Plugins using the older
-[`createSurfaceTexture`][] API will not work with [Impeller][].
+[`createSurfaceTexture`][] API will continue to work with [Impeller][], but are
```

Comment:
But only after the _next_ stable, we should probbly mention that caveat

Reply:
Done!